src/components/flat-table/flat-table.style.ts

```diff
@@ -364,6 +364,14 @@ StyledFlatTableWrapper.defaultProps = {
 const StyledFlatTableFooter = styled.div<
   Pick<FlatTableProps, "hasStickyFooter">
 >`
+  & > ${StyledPagerContainer} {
+    border-top: none;
+    border-top-width: 0;
```

Comment:
**suggestion** (non-blocking): I don't believe either `border-top-width: 0;` or `border-top-style: none;` are needed when we already have `border-top: none;`. And I find it clearer just to have the one style rather than 3!